lib/Newsletter/Editor/PostTransformer.php

```diff
@@ -76,20 +76,29 @@ private function getStructureWithLayout($post) {
         ))
       );
     }
+    $title_position = isset($this->args['titlePosition']) ? $this->args['titlePosition'] : '';
 
     if ($featured_image_position === 'aboveTitle' || $featured_image_position === 'belowTitle') {
       $featured_image_position = 'centered';
     }
 
     if ($featured_image_position === 'centered') {
-      array_unshift($content, $title, $featured_image);
+      if ($title_position !== 'aboveExcerpt') {
```

Comment:
Since it has both `if` and `else` it would probably make more sense not to use negative condition in the `if (...)`.